pkg/morph/client/notary.go

```diff
@@ -279,7 +281,7 @@ func (c *Client) notaryInvoke(committee bool, contract util.Uint160, method stri
 		0,
 		c.notary.fallbackTime,
 		c.acc)
-	if err != nil {
+	if err != nil && !alreadyOnChainError(err) {
```

Comment:
Add comment about the reason `already on chain` error is being ignored. For example:

```go
// Neo RPC node can return `core.ErrInvalidAttribute` error with
// `conflicting transaction <> is already on chain` message. This
// error is expected and ignored. As soon as main tx persisted on 
// chain everything is fine. This happens because notary contract 
// requires 5 out of 7 signatures to send main tx, thus last two 
// notary requests may be processed after main tx appeared on chain.
```